### DISCUSSION TITLE
[help] Chia stuck transactions!

### DISCUSSION THREAD

**Discussion by giang21199z** [Chia-Network/chia-blockchain#5039] [Q&A]
I'm tried to send 1 to MXC and set Amount 1 and Fees = 0.001.

I waited about 7 hours and transaction still not mined.
How I can cancel this transaction and resend it?
Please help me.

**Comment by cccat6** [comment]
You cannot cancel it. But is abnormal that didn't receive it for 7 hours.
We usually don't set fees for now, and you did. It should be much faster than the other transaction.
Check if you are sending to the correct receiving address, and search your transaction on www.chiaexplorer.com see what the status is.

**Reply by giang21199z** [reply]
Thanks for your response. 
Could you help for guide how to search my transaction on  www.chiaexplorer.com.
I checked by CLI my transaction. It's still pending.
![image](https://user-images.githubusercontent.com/11538834/118349973-4ef33300-b57e-11eb-8bde-441baf3b8f82.png)

**Reply by cccat6** [reply]
![image](https://user-images.githubusercontent.com/22387156/118349998-e78db100-b586-11eb-95e8-7804fea2f3e3.png)

**Reply by giang21199z** [reply]
Thanks a lot.
I need paste address wallet of chia network right?

**Reply by cccat6** [reply]
Any related information listed there should be fine

**Reply by giang21199z** [reply]
![image](https://user-images.githubusercontent.com/11538834/118350460-0f7a1600-b581-11eb-8ce2-552a8d199c1e.png)
This is my result when i search my address wallet of chia.

**Reply by giang21199z** [reply]
Has only two transactions farmer rewards. Don't have any tranction move XCH to market's wallet.

**Reply by cccat6** [reply]
I guess you didn't actually send the request to the network. There might have some problem with your local wallet. Maybe delete the wallet db and let it resync.
The file should be in `%userprofile%\.chia\mainnet\wallet\db`
